modules/fuzzel/hm.nix

```diff
@@ -13,7 +9,7 @@ in
   config.programs.fuzzel.settings =
     lib.mkIf (config.stylix.enable && config.stylix.targets.fuzzel.enable)
       {
-        colors = {
+        color = with config.lib.stylix.colors; {
```

Comment:
The setting name here has changed from `colors` to `color` - was this intentional?